tests/selenium/grievance/grievance_tickets/test_grievance_tickets.py

```diff
@@ -703,7 +703,7 @@ def test_grievance_tickets_create_new_ticket_Data_Change_Add_Individual_All_Fiel
         pageGrievanceNewTicket.getSelectIndividualdataSeeingdisability().click()
         pageGrievanceNewTicket.select_listbox_element("Some difficulty")
         pageGrievanceNewTicket.getSelectIndividualdataPhysicaldisability().click()
-        pageGrievanceNewTicket.select_listbox_element("None")
+        pageGrievanceNewTicket.select_listbox_element("Cannot do at all")
```

Comment:
shall we remove the line? since this should be nullable? @srugano

Reply:
Indeed we can remove it.